frescobaldi_app/extensions/__init__.py

```diff
@@ -791,28 +793,24 @@ def installed_extensions(self):
         result = list(self._infos.keys())
         return sorted(result)
 
-    def is_extension_exception(self, traceback):
+    def is_extension_exception(self, tb):
```

Comment:
question: Can't we keep "traceback"? That is a bit more verbose.

Reply:
Yes, of course. We can't use that now. I guess it's alright then. Or could you come up with something less vague?

Reply:
Another idea could perhaps be to `import extract_tb from traceback` to avoid the name collision!??